models/comments.model.js

```diff
@@ -19,4 +19,12 @@ exports.createCommentByArticleid = (article_id, obj) => {
     }).then((result) => {
         return result.rows[0]
     })
+}
+
+exports.removeComment = (comment_id) => {
+    return db.query("SELECT * FROM comments WHERE comment_id = $1;", [comment_id])
```

Comment:
Good logic, but this could be refactored. What would happen if you tried to delete a comment_id that didn't exist? Could you use that to determine if an id is valid or not?